### PR TITLE
HMAN-578 

### PR DESCRIPTION
**Pull request by padmapriyanalam**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/HMAN-578

### Change description ###

Audit Triage Information

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/hmc/client/featurehearing/ServiceBusMessageErrorHandlerTest.java [hunk 1]
```diff
@@ -17,12 +17,18 @@
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.powermock.api.mockito.PowerMockito;
```
Comment (markdathornehmcts) [inline src/test/java/uk/gov/hmcts/reform/hmc/client/featurehearing/ServiceBusMessageErrorHandlerTest.java:20]: Do you need PowerMockito?

File: src/main/java/uk/gov/hmcts/reform/hmc/Application.java [hunk 2]
```diff
@@ -3,14 +3,13 @@
...
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 import org.springframework.cloud.openfeign.EnableFeignClients;
 import org.springframework.context.annotation.Bean;
 import uk.gov.hmcts.reform.hmc.config.MessageReceiverConfiguration;
 
 import java.time.Clock;
 
-@SpringBootApplication(exclude = {DataSourceAutoConfiguration.class })
```
Comment (markdathornehmcts) [inline src/main/java/uk/gov/hmcts/reform/hmc/Application.java:13]: Good reason for this change? Just a query.

**Review by markdathornehmcts — APPROVED** [review]
Just a couple of suggestions and query.

File: src/main/java/uk/gov/hmcts/reform/hmc/Application.java (hunk 2 above)
Comment (padmapriyanalam) [reply src/main/java/uk/gov/hmcts/reform/hmc/Application.java:13]: To enable database auto-configuration in Spring Boot removed the exclude condition. HMAN-578 requires Outbound to insert the hearing journey into database.

**Review by markdathornehmcts — APPROVED** [review]
LGTM

**Pull request merged**